main.go

```diff
@@ -71,7 +71,7 @@ func main() {
 		options.CtrlConfig.MaxConcurrentReconciles = 1
 	}
 
-	ctrl.SetLogger(zap.Logger(true))
+	ctrl.SetLogger(zap.New(func(o *zap.Options) { o.Development = true }))
```

Comment:
this change is not needed ?

Reply:
now a SetLogger() requires a functional option